plugin/action/mask/mask.go

```diff
@@ -73,6 +81,12 @@ type Config struct {
 	// >
 	MaskAppliedValue string `json:"mask_applied_value"` // *
 
+	// > @3@4@5@6
+	// >
+	// > List of the ignored event fields.
+	IgnoreFields   []string `json:"ignore_fields"` // *
+	IgnoredFields_ map[string]interface{}
```

Comment:
It would be better to move this map to `Plugin` and rename to `ignoredFields`. Config fields with names `SomeField_` are usually used to store parsed values.